Mathlib/Topology/UniformSpace/Cauchy.lean

```diff
@@ -310,6 +310,17 @@ theorem tendsto_nhds_of_cauchySeq_of_subseq [Preorder β] {u : β → α} (hu :
   le_nhds_of_cauchy_adhp hu (mapClusterPt_of_comp hf ha)
 #align tendsto_nhds_of_cauchy_seq_of_subseq tendsto_nhds_of_cauchySeq_of_subseq
 
+/-- Any shift of a Cauchy sequence is also a Cauchy sequence. -/
+theorem cauchySeq_shift {u : ℕ → α} (k : ℕ) : CauchySeq u ↔ CauchySeq fun n ↦ u (n + k) := by
```

Comment:
Could you exchange the two sides of the iff? A general rule of thumb is that the more complicated expression should be on the left of the iff.

Reply:
Done.